### ISSUE TITLE
Improve libcosmic docs (?)

### ISSUE THREAD

**Issue by elevenhsoft** [pop-os/libcosmic#602]
Hello,

Right now we have docs located there: https://pop-os.github.io/libcosmic/cosmic/index.html and it's fine (except slow searching ;)). I'm not familiar with generating documentation, but I see there is for example no info about applet, which is hidden by feature. Is it possible to improve this docs, and include variables and functions from features (i guess that's the issue). I was looking for applets, and there is no info about it. Also if we look into [Core](https://pop-os.github.io/libcosmic/cosmic/app/struct.Core.html) for example, we see in `source` there is public field `applet` hidden by feature, so on the main page it's not showing in public fields, but it says the rest are private.

**Comment by mmstick** [comment]
They're automatically generated from code comments above functions and types; and at the top of modules! with `cargo doc`.

**Comment by elevenhsoft** [comment]
Yeah I was not sure since I never used it :sweat_smile: On docs.rs we have some more info, about features etc so I though it's something missing.

So we should use source code probably in this case.

**Comment by wiiznokes** [comment]
Unfortunately, `winit` and `applet` features are not additive, so we can't call `cargo doc --no-deps --all-features`. A solution for this could be to add another crates specific to `applets`
